plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1539,8 +1538,10 @@ producer_plugin_impl::start_block_result producer_plugin_impl::start_block() {
                         fc_dlog(_trx_trace_log, "[TRX_TRACE] Node with producers configured is dropping an EXPIRED transaction that was PREVIOUSLY ACCEPTED : ${txid}",
                                ("txid", trx->id()));
                      }
-                     itr = unapplied_trxs.erase( itr ); // unapplied_trxs map has not been modified, so simply erase and continue
-                     continue;
+                     if( preprocess_deadline <= fc::time_point::now() ) exhausted = true;
```

Comment:
Shouldn't there be a `++num_processed; ++num_failed;` here as well?

I know it wasn't there in the original code either, but it seems like it would be under-reporting the number of transactions that were processed and dropped.

Reply:
Seems like it. Added.